### PR TITLE
Add missing dependency

### PR DESCRIPTION
**Pull request by szepeviktor**
Used in inc/upgrades/namespace.php five times.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
 		"aws/aws-sdk-php": "^3.224.3",
 		"composer-plugin-api": "^1.1 || ^2.0",
 		"composer/installers": "^1.12 || ^2.0",
+		"humanmade/clean-html": "^2.0",
```
Comment (roborourke) [inline composer.json:17]:
```suggestion
		"humanmade/clean-html": "^2.0.0",
```

**Review by roborourke — COMMENTED** [review]
It's a dependency of `altis/cms` currently. Are you using a subset of Altis modules?

**Comment by szepeviktor** [comment]
> Are you using a subset of Altis modules?

I'm not a user. This is a random contribution.

**Comment by roborourke** [comment]
Ok cool, well thank you!

**Comment by mikelittle** [comment]
This is already included indirectly by composer dependencies. No need to include it directly. Thanks.

**Pull request closed without merging**